gpt_index/readers/file.py

```diff
@@ -99,7 +118,10 @@ def _add_files(self, input_dir: Path) -> List[Path]:
             new_input_files.extend(sub_input_files)
 
         if self.num_files_limit is not None and self.num_files_limit > 0:
-            new_input_files = new_input_files[0 : self.num_files_limit]
+            new_input_files = new_input_files[0: self.num_files_limit]
+
+        # print total number of files added
+        print(f"> [SimpleDirectoryReader] Total files added: {len(new_input_files)}")
```

Comment:
i haven't followed this rule up to now, but i want to put most print statements behind a `verbose` option. may be good to add a `verbose` option during the `__init__` of the BaseReader (or remove for now).

Reply:
Made the change - I think this is what you mean?